src/StepFactory.php

```diff
@@ -32,11 +32,21 @@ public static function create(array $step)
     /**
      * @param array $step
      *
+     * @return array
+     */
+    private static function getEntitiesFromStepData(array $step = [])
+    {
+        return array_key_exists('entities', $step) ? $step['entities'] : [];
```

Comment:
Absolutely, we should do that. If wit.ai does everything right it will probably never be empty, but we shouldn't rely on a third party if we don't have to.